Makefile

```diff
@@ -33,12 +33,16 @@ VERSION ?= master
 
 TAG_LATEST ?= false
 
+# The version of restic binary to be downloaded for power architecture
+RESTIC_VERSION ?= 0.9.4
+
 ###
 ### These variables should not need tweaking.
 ###
 
 CLI_PLATFORMS := linux-amd64 linux-arm linux-arm64 darwin-amd64 windows-amd64 linux-ppc64le
-CONTAINER_PLATFORMS := linux-amd64 linux-arm linux-arm64 linux-ppc64le
+CONTAINER_PLATFORMS := linux-amd64 linux-ppc64le #linux-arm linux-arm64
```

Comment:
Why are arm platforms removed?

Reply:
arm is commented out of CONTAINER_PLATFORMS as there are no images being maintained for them at the moment. The cross-compile binaries will be built for those architectures as they are part of CLI_PLATFORMS. Also, the current workflow is building images explicitly for linux-amd64 so the arm platforms are not utilized for image building for now.